plugins/sqs/include/operator.hpp

```diff
@@ -0,0 +1,274 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2024 The Tenzir Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include <tenzir/detail/env.hpp>
+#include <tenzir/plugin.hpp>
+#include <tenzir/series_builder.hpp>
+
+#include <aws/core/Aws.h>
+#include <aws/core/utils/Outcome.h>
+#include <aws/sqs/SQSClient.h>
+#include <aws/sqs/model/CreateQueueRequest.h>
+#include <aws/sqs/model/DeleteMessageRequest.h>
+#include <aws/sqs/model/GetQueueUrlRequest.h>
+#include <aws/sqs/model/ReceiveMessageRequest.h>
+#include <aws/sqs/model/SendMessageRequest.h>
+
+using namespace std::chrono_literals;
+
+namespace tenzir::plugins::sqs {
+
+namespace {
+
+/// The default poll time.
+static constexpr auto default_poll_time = 3s;
```

Comment:
The docs say that this is `10s`. They also say this for the old TQL1 connector though. 

I wouldnt want to change the actual default value, so it would be best to update the docs, including the past ones.